web/ui/react-app/src/Navbar.tsx

```diff
@@ -25,6 +34,11 @@ const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
       </Link>
       <Collapse isOpen={isOpen} navbar style={{ justifyContent: 'space-between' }}>
         <Nav className="ml-0" navbar>
+          {consoleLink !== '' && (
+            <NavItem>
+              <NavLink href={`${pathPrefix + consoleLink}`}>Consoles</NavLink>
```

Comment:
The consoles link already includes the path prefix, so `pathPrefix` should not be needed here.